bin/filter.rb

```diff
@@ -24,6 +24,13 @@ def process(data, force_header: nil, format_for_frontpage: false)
 
   doc = Nokogiri::HTML::DocumentFragment.parse(body)
 
+  # Grab footer image for social media previews
```

Comment:
Maybe only do this when `format_for_frontpage` is `false`?

Reply:
good call.